packages/driver/cypress/e2e/commands/waiting.cy.js

```diff
@@ -737,6 +737,83 @@ describe('src/cy/commands/waiting', () => {
       })
     })
 
+    describe('multiple alias execution order', () => {
```

Comment:
Suggest removing this `describe` block and just moving the tests under `describe('multiple alias arguments',`

Reply:
Updated.